docs/sources/developers/plugins/migration-guide.md

```diff
@@ -48,6 +50,81 @@ Removed due to being an internal component.
 
 Removed due to being an internal component.
 
+### 8.3 deprecations
+
+#### LocationService replaces getLocationSrv
+
+A while back we migrated to use a new routing system. At the same time we introduced a new service for managing locations, navigation etc. and we are now making that service the primary one to use.
+
+**Example:** How to import the service.
+
+```ts
+// before
+import { getLocationSrv } from '@grafana/runtime';
+
+// after
+import { locationService } from '@grafana/runtime';
+```
+
+**Example:** How to navigate to a path and add a new record in the navigation history so you can navigate back to the previous one.
+
+```ts
+// before
+getLocationSrv.update({
+  path: '/route-to-navigate-to',
+  replace: false,
+});
+
+// after
+locationService.push('/route-to-navigate-to');
+```
+
+**Example:** How to navigate to a path and replacing the current record in the navigation history.
+
+```ts
+// before
+getLocationSrv.update({
+  path: '/route-to-navigate-to',
+  replace: true,
+});
+
+// after
+locationService.replace('/route-to-navigate-to');
+```
+
+**Example:** How to update the search/query parameter for the current route and add a new record in the navigation history so you can navigate back to the previous one.
```

Comment:
```suggestion
**Example:** Update the search or query parameter for the current route and add a new record in the navigation history so that you can navigate back to the previous one.
```